app/views/forgot_passwords/new.html.haml

```diff
@@ -0,0 +1,10 @@
+%section.forgot_password.container
+  .row
+    .col-sm-10.col-bg-offset-1
+      = bootstrap_form_tag url: forgot_passwords_path, class: 'sign-in', layout: :horizontal, control_col: 'col-sm-6' do |f|
+        %header
+          %h1 Forgot Password?
+          %p We will send you an email with a link that you can use to reset your password.
+        .form-group
```

Comment:
`bootstrap_form_for` generates `form-group` elements for you. You don't need to create this one yourself.